.ci_support/win_64_numpy1.18python3.7.____cpython.yaml

```diff
@@ -1,5 +1,5 @@
 channel_sources:
-- conda-forge,defaults
```

Comment:
@beckermr @jaimergp Thanks for helping look at this PR.  Note that I see here that the `defaults` channel is removed.  Is it possible that the `defaults` channel `fortran` compiler was found differently than the `conda-forge` one?
@fsciortino Maybe add it back in manually here?

Reply:
We use strict priority so the compilers are coming from conda-forge in all cases. defaults was only there to cover for missing packages in conda-forge, which is no longer the case.